packages/jellyfish-transaction-builder/src/txn/txn_builder_account.ts

```diff
@@ -0,0 +1,85 @@
+import BigNumber from 'bignumber.js'
+import { DeFiTransactionConstants, OP_CODES, Script, Transaction, TransactionSegWit, Vout } from '@defichain/jellyfish-transaction'
+import { UtxosToAccount } from '@defichain/jellyfish-transaction/dist/script/defi/dftx_account'
+import { P2WPKHTxnBuilder } from './txn_builder'
+import { TxnBuilderError, TxnBuilderErrorType } from './txn_builder_error'
+
+export class TxnBuilderAccount extends P2WPKHTxnBuilder {
+  /**
+   * Requires at least 0.001 DFI to create transaction, actual fees are much lower.
+   *
+   * @param {BigNumber} amount amount of utxo to be converted
+   * @param {Script} toScript to hold converted token and to send unspent to after deducting the (converted + fees)
+   */
+  async utxosToAccount (amount: BigNumber, toScript: Script): Promise<TransactionSegWit>
+
+  /**
+   * Requires at least 0.001 DFI to create transaction, actual fees are much lower.
+   *
+   * @param {UtxosToAccount} utxosToAccount txn to create
+   * @param {Script} changeScript to send unspent to after deducting the (converted + fees)
+   */
+  async utxosToAccount (utxosToAccount: UtxosToAccount, changeScript: Script): Promise<TransactionSegWit>
+
+  async utxosToAccount (arg: BigNumber | UtxosToAccount, toScript: Script): Promise<TransactionSegWit> {
+    let utxosToAccount: UtxosToAccount
+
+    if (BigNumber.isBigNumber(arg)) {
+      utxosToAccount = {
+        to: [{
+          balances: [{
+            token: 0,
+            amount: arg
+          }],
+          script: toScript
```

Comment:
Few things I am uncertain here

- utxosToAccount only applicable to DFI
- store the converted token together with change in single redeem script